Document SanitizeOnShutdown Exceptions (Firefox 154) (#237)
mozilla/enterprise-admin-reference: src/content/docs/reference/policies/Cookies.mdx

## Values

- `Allow` is a list of origins (not domains) where cookies are always allowed. You must include `http` or `https`.

>   As of Firefox 154, using `Allow` to also exempt sites from clearing on shutdown is deprecated. Firefox migrates those origins to the new mechanism, but you should configure [`SanitizeOnShutdown`](/reference/policies/sanitizeonshutdown/)'s `Exceptions` instead.

Reviewer: Looks good, minor suggestion:

```suggestion
  As of Firefox 154, using `Allow` to exempt sites from clearing on shutdown is deprecated. Firefox migrates origins to the new mechanism, but you should use [`SanitizeOnShutdown`](/reference/policies/sanitizeonshutdown/)'s `Exceptions` instead.
```
